config/config.go

```diff
@@ -97,10 +103,6 @@ type Config struct {
 	DestinationBlockchains []*DestinationBlockchain `mapstructure:"destination-blockchains" json:"destination-blockchains"`
 	ProcessMissedBlocks    bool                     `mapstructure:"process-missed-blocks" json:"process-missed-blocks"`
 	ManualWarpMessages     []*ManualWarpMessage     `mapstructure:"manual-warp-messages" json:"manual-warp-messages"`
-
-	// convenience fields to access the source subnet and chain IDs after initialization
-	sourceSubnetIDs     []ids.ID
-	sourceBlockchainIDs []ids.ID
```

Comment:
These were only used to pass to the app request network constructor. With this change, it makes more sense to pass the configuration and iterate over the `SourceBlockchain` and `DestinationBlockchain` slices directly, so the parsed IDs were moved there.